eng/pipelines/jobs/platform-matrix.yml

```diff
@@ -0,0 +1,149 @@
+# Executes a specified job on each supported platform and configuration
+
+parameters:
+  # Job template to run on applicable platforms
+  jobTemplate: ''
```

Comment:
Any notable changes in the matrix?

Reply:
Nothing changed in the matrix and all of the effective parameters are the same.